src/components/AppFooter.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+
+import VStack from '@/components/lib/VStack/VStack.vue'
+import constants from '@/utils/constants'
+
+const route = useRoute()
+
+const canShowDocs = computed(() => {
+  return route.name === 'Login'
+})
+</script>
+
+<template>
+  <footer>
+    <VStack justify="space-between" class="container">
+      <VStack gap="2.5rem" align="center">
```

Comment:
The alignment here is visually off. The middle column is not vertically centred to the screen. You can see the shift in the placement of the "Powered by Arcana" text in the footer between the landing page and the other pages.

```
[lhs] [mid] [rhs]
```

This is happening because lhs and rhs have different widths and `justify-content: space-between` distributes space equally between columns. One way we can fix it is by making sure lhs and rhs take equal widths. We can do that by setting:

- the lhs group to `flex: 1`
- the rhs group to `flex: 1` and `justify-content: end`
- remove the `justify-content: space-between` from the parent.

This will always keep the mid group centered. [See this example](https://jsbin.com/soqolivoxe/edit?html,output) on jsbin for details.

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/109/commits/e6fbd93c3a5e2f530e7f944af579485e333d8576